### PR TITLE
Added missing `pt-BR` translations

### PR DESCRIPTION
**Pull request by marcosmartini**
# Explain your changes

Added missing `pt-BR` translations.

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by marcosmartini** [comment]
@DaveOrDead Will fix it

**Comment by marcosmartini** [comment]
@DaveOrDead @evgenyk Updated `pt-BR`

File: pt-BR/auth.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "spec": {
-    "language": "pt-BR",
+    "language": "en",
```
Comment (evgenyk) [inline pt-BR/auth.json:3]: @marcosmartini it looks like the language is incorrectly changed to `en`

Comment (marcosmartini) [reply pt-BR/auth.json]: @evgenyk Sorry, updated now

**Pull request merged**